sentinelone/plugin.spec.yaml

```diff
@@ -3,7 +3,7 @@ extension: plugin
 products: [insightconnect]
 name: sentinelone
 title: SentinelOne
-version: 5.0.1
+version: 5.1.0
```

Comment:
This will need a major version bump with the type changes 👍 

e.g. 

```
- type: object
+ type: active_directory
```

Reply:
Thanks, updated the version to 6.0.0 🙂